go.mod

```diff
@@ -1,15 +1,37 @@
 module github.com/mainflux/export
 
-go 1.14
+go 1.20
 
 require (
-	github.com/eclipse/paho.mqtt.golang v1.2.0
-	github.com/go-kit/kit v0.10.0
-	github.com/go-redis/redis v6.15.8+incompatible
+	github.com/eclipse/paho.mqtt.golang v1.4.2
```

Comment:
PLease check for the latest version.

Reply:
this seems to  be the latest https://github.com/eclipse/paho.mqtt.golang/releases